registry/storage/driver/s3-aws/s3.go

```diff
@@ -261,6 +268,7 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		fmt.Sprint(rootDirectory),
 		storageClass,
 		fmt.Sprint(userAgent),
+		fmt.Sprint(objectAcl),
```

Comment:
Can we test this is equal to a valid value as defined in vendor/github.com/aws/aws-sdk-go/service/s3/api.go?